src/main/java/controller/GetMessage.java

```diff
@@ -46,6 +48,9 @@
     @Autowired
     private MessageAccessor queryMessage;
 
+    @Autowired
+    private AttachmentAccessor queryAttachment;
```

Comment:
Let's call it `attachmentAccessor`.

Reply:
It will be different from the other `Autowired` members used to access `Spanner` relations. Is that okay?

Reply:
Sure. That is fine.